locales/en.json

```diff
@@ -1,111 +1,113 @@
 {
-    "about": {
```

Comment:
Can you please re-indent this file at 4 spaces so that the diff is only the new changes?

`eslint --fix **/*.json` 🤔 it might work, i am not sure

The python tool will definitely work:

```sh
cd tools
# macOS uses python3, on windows / linux python3 should just be python
python3 translation_manager.py alphabetize
```